src/app/components/Headings/index.jsx

```diff
@@ -28,7 +28,14 @@ export const Headline = styled.h1`
   }
 `;
 
-export const SubHeading = styled.h2`
+const regexPunctuationSymbols = /[.,/#!$%^&*;:{}=\-_~()'"]/g;
```

Comment:
It would be better to do a negated regex match rather than explicitly mentioning those we don't want. Also, we want to keep dashes.